Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs

```diff
@@ -2,14 +2,24 @@
 // Licensed under the MIT License. See LICENSE in the project root for
 // license information. 
 
+using Microsoft.Extensions.PlatformAbstractions;
+using System;
+using System.Reflection;
+
 namespace Artesian.SDK.Service
 {
     /// <summary>
     /// Artesian Constants
     /// </summary>
     public abstract class ArtesianConstants
     {
-        internal const string CharacterValidatorRegEx = @"^[^'"",:;\s](?:(?:[^'"",:;\s]| )*[^'"",:;\s])?$";
+        internal const string SDKVersion = "v2.2.1";
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+		public static string SDKVersionHeaderValue = $@".NET<{Assembly.GetExecutingAssembly().GetName().Version}>,{Environment.OSVersion.Platform}<{Environment.OSVersion.Version}>,{PlatformServices.Default.Application.RuntimeFramework.Identifier}<{PlatformServices.Default.Application.RuntimeFramework.Version}>";
```

Comment:
the format is not with <> afaik, ask Alessandro. It was changed during development, but he probably forgot to update the Task/Requirement specification. @sicurezza

Reply:
also is missing 'readonly' specifier

Reply:
Thanks, now I have the right version